CHANGELOG.rst

```diff
@@ -16,3 +16,9 @@ Changelog
 * Fixed empty string bug on git version of KMS user agent
 * Local tests only by default
 * Fix of MAP_ANONYMOUS issue for older Linuxes
+
+0.1.3 -- 2019-04-29
+===================
+* Added API function to make session from keyring
+* Added API function to make caching CMM from keyring
+* Added "_from_cmm" to end of "aws_cryptosdk_caching_cmm_new" function name
```

Comment:
We are removing an API surface. This is a breaking change, and thus per our verisoning policy this must be a minor version bump (ie: `0.2.0`).

Reply:
Okay, I didn't think these issues necessarily applied to the pre 1.0.0 versions, but it's no big deal to switch this to 0.2.0. I will make that change and then merge the PR.

Reply:
Ah, @mattsb42-aws I noticed that you clicked the button to block the merge on your requested change, so I will let you review the version bump. Please feel free to merge this code yourself if everything is cool.

I decided that it was a good time to get one more name change in, since we're making a minor version bump anyway and it's better to get things stable before we put out the 1.0.0 release.

So I'm adding another commit to change the function ``aws_cryptosdk_session_get_algorithm`` to ``aws_cryptosdk_session_get_alg_id``. This is more shorter, more accurate, and more consistent with the other public API function we have that refers to algorithm IDs, i.e., ``aws_cryptosdk_default_cmm_set_alg_id``

Everything else is the same as the code that @ttjsu-aws already approved.